packages/store/schematics/utils/common/lib.config.ts

```diff
@@ -1,28 +1,63 @@
 import { NodeDependency, NodeDependencyType } from '@schematics/angular/utility/dependencies';
 
 export enum LIBRARIES {
-  STORE = '@ngxs/store',
+  DEVTOOLS = '@ngxs/devtools-plugin',
+  FORM = '@ngxs/form-plugin',
+  HMR = '@ngxs/hmr-plugin',
   LOGGER = '@ngxs/logger-plugin',
-  DEVTOOLS = '@ngxs/devtools-plugin'
+  ROUTER = '@ngxs/router-plugin',
+  STORAGE = '@ngxs/storage-plugin',
+  STORE = '@ngxs/store',
+  WEBSOCKET = '@ngxs/websocket-plugin'
 }
 
 export const LIB_CONFIG: NodeDependency[] = [
   {
     type: NodeDependencyType.Default,
     name: LIBRARIES.DEVTOOLS,
-    version: '^3.3.2',
+    version: '3.7.3',
```

Comment:
The common version number across all of these should probably best be refactored to a shared variable.

Reply:
@markwhitfeld updated, review again please.